### DISCUSSION TITLE
Dual dialogue visualization

### DISCUSSION THREAD

**Discussion by pluntert** [chuangcaleb/obsidian-fountain-editor#34] [Ideas]
First of all: thanks for this amazing plugin! Now to the idea/feature request:

It would be great to have a visualization of the [dual dialogue feature](https://fountain.io/syntax/#dual-dialogue). Would something like that be possible to implement?

**Comment by chuangcaleb** [comment]
Welcome! Dual dialogue is already a tracked task in the [project board](https://github.com/users/chuangcaleb/projects/2/views/6?pane=issue&itemId=41603278). 

But as you can see, it's in the SEND HELP board. I have no idea how to implement this — side by side blocks of text may need some hacking to work with HTML's usual single-column flow. It may also raise conflict with the current styling mechanics, and may require an entire styling overhaul to get dual dialogue working. 

You can still write amazing Fountain without it (I've read somewhere that it may even be better to not have it, as it breaks conventional single-column flow?), so all this is why it's in the "nice to have while tough to add" category. But if someone would like to take a shot at it, please open a Pull Request!

**Comment by pluntert** [comment]
Thanks for the quick response and explanation! And sorry for not taking a peek at the project board first! I'm not that familiar with Github's structure yet.

It is definitely not the most necessary feature, and it is already possible to just put a caret next to a character with an at sign right before it, so the option is there if someone would want to use the dual dialogue feature without the visualization.

Unfortunately, I have no programming knowledge, so taking a shot at it is beyond my skillset.

Anyway, many thanks for the amazing plugin again!
